aretomo/protocols/protocol_aretomo.py

```diff
@@ -631,6 +739,18 @@ def getOutputSetOfTiltSeries(self,
                                        outputSetOfTiltSeries)
         return getattr(self, outputName)
 
+    def getOutputSetOfCtfs(self) -> SetOfCTFTomoSeries:
+        outputCtfs = getattr(self, OUT_CTFS, None)
+        if outputCtfs:
+            outputCtfs.enableAppend()
+        else:
+            outputCtfs = SetOfCTFTomoSeries.create(self._getPath(), template='CTFmodels%s.sqlite')
+            outputCtfs.setSetOfTiltSeries(self.getOutputSetOfTiltSeries(OUT_TS))
```

Comment:
what is the difference between newCTFTomoSeries.setTiltSeries(newTs) and outputCtfs.setSetOfTiltSeries(..)?

Reply:
I got it, the outputCtfs is the set object.. :) Maybe we should rename methods to match..